scripts/rust-analyze.sh

```diff
@@ -12,10 +12,11 @@ set -x # Show commands
 CONFIG_FILE=$(realpath $1)
 TREE_NAME=$2
 
-find $OBJDIR -type d -name save-analysis |
-while read ANALYSIS_DIR; do
+if [ -e "$OBJDIR" ]; then
```

Comment:
This should probably be -d instead of -e to ensure it's a directory and not just a file

Reply:
Yup, agreed. Done